### DISCUSSION TITLE
How to change the column width?

### DISCUSSION THREAD

**Discussion by pastoraldr** [lxndrblz/anatole#377] [Q&A]
The layout is great. But I wonder if I can change the width of the first column

**Comment by lxndrblz** [comment]
@pastoraldr this can easily be done, as explained here:

https://github.com/lxndrblz/anatole/wiki/%F0%9F%92%85-Appearance#adjusting-the-sidebarcontent-ratio

**Reply by pastoraldr** [reply]
Thanks a lot, Alex.

**Reply by pastoraldr** [reply]
Hi, Alex, sorry to bother you again. I have another question about changing the mini-picture on the tab of browsers. I was able to change the profile photo in params.toml.

**Reply by lxndrblz** [reply]
@pastoraldr In your params you need to set the `favicon = "favicons/" like this. Then create a favicons folder inside of the static folder. The theme will automatically look for a favicon.ico file.
